config/jobs/kubernetes/sig-testing/dependencies.yaml

```diff
@@ -30,6 +30,36 @@ presubmits:
         - name: WHAT
           value: "vendor vendor-licenses"
 
+  - name: pull-kubernetes-dependencies-canary
+    path_alias: "k8s.io/kubernetes"
+    decorate: true
+    branches:
+    - master
+    always_run: true
+    skip_report: true
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - name: main
+        command:
+        - runner.sh
+        image: gcr.io/k8s-testimages/kubekins-e2e:v20190420-93fab49-master
+        args:
+        - make
+        - verify
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+        env:
+        # Space separated list of the checks to run
+        - name: WHAT
+          value: "vendor vendor-licenses"
+        - name: GOPROXY
+          value: https://proxy.golang.org
+        - name: GOSUMDB
+          value: sum.golang.org
```

Comment:
I think this won't take affect with our go version, but can't hurt to have.
https://groups.google.com/forum/#!msg/golang-dev/go6_lZqew6g/rNg3KacMDQAJ

Reply:
(GOSUMDB that is)